VERSION

```diff
@@ -0,0 +1 @@
+0.0.1.dev
```

Comment:
need to discuss versioning

Reply:
set to 1.27.2. Not fully up to date with original Z2MQTT repo (now on 1.28.2). Once we merge 1.27 branch into master later I will then update master from original repo.